typescript/utils/src/addresses.ts

```diff
@@ -372,3 +372,8 @@ export function ensure0x(hexstr: string) {
 export function strip0x(hexstr: string) {
   return hexstr.startsWith('0x') ? hexstr.slice(2) : hexstr;
 }
+
+export const fromHexString = (hexstr: string) =>
+  Buffer.from(strip0x(hexstr), 'hex');
+
+export const toHexString = (buf: Buffer) => ensure0x(buf.toString('hex'));
```

Comment:
redundant with `bytesTo*` methods above

Reply:
ditto, I dont think they are redundant

Reply:
already been moved to strings.ts in util